AsyncDisplayKit/Details/ASBatchContext.h

```diff
@@ -59,6 +56,20 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)beginBatchFetching;
 
+
+#pragma mark - Deprecated
+
+/**
+ * Let the context object know that a batch fetch was completed.
+ *
+ * @param didComplete A boolean that states whether or not the batch fetch completed.
+ *
+ * @discussion Only by passing YES will the owner of the context know to attempt another batch update when necessary.
+ * For instance, when a table has reached the end of its data, a batch fetch will be attempted unless the context
+ * object thinks that it is still fetching.
+ */
+- (void)completeBatchFetching:(BOOL)didComplete;
```

Comment:
I revert the code for now. In my opinion would be good to revisit the the `-[ASBatchContext completeBatchFetching:]` API e.g. Passing in `NO` is basically a no op.